src/test/java/com/cloudbees/plugins/credentials/CredentialsProviderTest.java

```diff
@@ -59,7 +68,7 @@
 
     @Rule
     public JenkinsRule r = new JenkinsRule();
-    
```

Comment:
Traditionally we fix whitespace and other formatting changes in a dedicated PR. Could you move those changes to a different PR please

Reply:
Thanks for the feedback. I think this was because Atom trimmed blank lines when I edited the file. Are you happy if I push another commit to revery these, or would you like me to squash all the ones I've made so far and re-push to my branch?